reference/customers.v2.yml

```diff
@@ -1220,32 +1220,107 @@ components:
-                example: "5.0000"
-              price_list_id:
-                type: integer
-                description: If a customer group is assigned to a price list,`method` and `amount` are not shown. `type` and `price_list_id` are returned.
-                example: 3
+          oneOf:
+          - type: array
+            description: A collection of discount rules that are automatically applied to customers who are members of the group.
+            items:
+              type: object
+              properties:
+                type:
+                  type: string
+                  enum:
+                    - price_list
+                    - all
+                    - category
+                    - product
+                method:
+                  type: string
+                  enum:
+                    - percent
+                    - fixed
+                    - price
+                amount:
+                  type: string
+                  description: A float that specifies the value applied to the price modified. (Float, Float as String, Integer)
+                  example: "5.0000"
+                price_list_id:
+                  type: integer
+                  description: If a customer group is assigned to a price list,`method` and `amount` are not shown. `type` and `price_list_id` are returned.
+                  example: 3
+          - type: array
+            description: A collection of category-level discount rules that are automatically applied to customers who are members of the group.
+            items:
+              type: object
+              properties:
+                type:
+                  type: string
+                  enum:
+                    - price_list
+                    - all
+                    - category
+                    - product
+                method:
+                  type: string
+                  enum:
+                    - percent
+                    - fixed
+                    - price
+                amount:
+                  type: string
+                  description: A float that specifies the value applied to the price modified. (Float, Float as String, Integer)
+                  example: "5.0000"
+                category_id:
+                  type: integer
+                  description: If a customer group is assigned to a category, `method` and `amount` are not shown. `type` and `category_id` are returned.
+                  example: 3
+          - type: array
+            description: A collection of product-level discount rules that are automatically applied to customers who are members of the group.
+            items:
+              type: object
+              properties:
+                type:
+                  type: string
+                  enum:
+                    - price_list
+                    - all
+                    - category
+                    - product
```

Comment:
Similar with the above comment, since we have product_id in this array, then I think the type here would have to be product?